scripts/project-setup-scripts/terra-billing-project-owner.yaml

```diff
@@ -2,16 +2,21 @@ title: "terra_billing_project_owner"
 description: "permissions for billing project owners"
 stage: "GA"
 includedPermissions:
-# <viewer> Start of permissions included as part of roles/viewer
```

Comment:
drive by review: I would leave this in. It is explaining why the following permissions were added (because they're part of roles/viewer role)

Reply:
The reason why I pulled it out was because I didn't think it was all too helpful.  It's not wrong, but the `Viewer` role is so broad, it's not really helping us understand why we need these specific permissions in our custom role.  I would be happy to put it back in if you feel strongly though.

Reply:
Maybe we say that these permissions were the ones from the roles/viewer role that we thought were important to keep? I don't feel strongly about keeping or removing it though.